documentation/RLC-quickstart.md

```diff
@@ -16,96 +14,100 @@ If you are the first time to prepare the SDK, please follow the Azure SDK guidan
-    
-    ```yaml
-    emit:
-      - "@azure-tools/typespec-ts"
-    options:
-      "@azure-tools/typespec-ts":
-        title: Farmbeats
-        generateMetadata: true
-        generateTest: true
-        "emitter-output-dir": "{output-dir}"
-        packageDetails:
-          name: "@azure-rest/agrifood-farming"
-          description: "Farmbeats Client"
-          version: "1.0.0-beta.1"
+Also we have some other options, you could refer to [the link](https://github.com/Azure/autorest.typescript/tree/main/packages/typespec-ts#emitter-options) for more details.
 
-    ```
+2. **Configure tsp-location.yaml**
 
-    Here, we need to replace the value in `name`,`description`, `version` in `packageDetails` to **your own service's** package details. Also we have some other options, you could refer to [the link](https://github.com/Azure/autorest.typescript/tree/main/packages/typespec-ts#emitter-options) for more details.
+Make sure there are tsp-location.yaml under your local emitter-output-dir. Example:
 
-    ---  
-    **NOTE**
+```yaml
+cleanup: false
+commit: 75ece9197dbac70ac0ba651c53a79c1841944be2
+directory: specification/cognitiveservices/AnomalyDetector
+repo: Azure/azure-rest-api-specs
+```
 
-    After the first generation, you need to switch `generateMetadata: false` as we have some manual changes in this file and don't want them get overwrite by generated ones.
+3. Run cmd to generate the SDK
 
-    --- 
+Step into root folder of SDK repo and replace `YOUR_TARGET_SDK_FOLDER` with your real SDK folder (like "sdk\anomalydetector\ai-anomalydetector-rest"), then run:
 
-3. **Edit rush.json**  
-    As the libraries in azure-sdk-for-js repository are managed by rush, you need to add an entry in rush.json under projects section for the first time to make sure it works. For example:
+```
+PS D:\dev\azure-sdk-for-js> pwsh eng\common\scripts\TypeSpec-Project-Sync.ps1 YOUR_TARGET_SDK_FOLDER
+PS D:\dev\azure-sdk-for-js> pwsh eng\common\scripts\TypeSpec-Project-Generate.ps1 YOUR_TARGET_SDK_FOLDER
+```
+
+---
+
+**NOTE**
 
-    ```
-        {
-          "packageName": "@azure-rest/agrifood-farming",
-          "projectFolder": "sdk/agrifood/agrifood-farming-rest",
-          "versionPolicyName": "client"
-        },
-    ```
+After the first generation, you need to switch `generateMetadata: false` as we have some manual changes in this file and don't want them get overwrite by generated ones.
 
-    Here you also need to replace the `packageName`, `projectFolder` into your own services'.
+---
 
-    ---  
-    **NOTE**
+3. **Edit rush.json**  
+   As the libraries in azure-sdk-for-js repository are managed by rush, you need to add an entry in rush.json under projects section for the first time to make sure it works. For example:
 
-    About the `versionPolicyName`, if the library you are working on is for data-plane, then it should be `client`, if the library you are working on is for control plane, then it should be `mgmt`.  
+   ```
+       {
```

Comment:
As we are now using script inside the JS repo to generate the SDK, can this step also be done automatically?  We could check the package name pattern (contain arm-) to decide whether this is mgmt plane or data plane.